packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-classes/options.json

```diff
@@ -1,3 +1,4 @@
 {
-  "plugins": [["proposal-decorators", { "version": "2023-11" }]]
+  "plugins": [["proposal-decorators", { "version": "2023-11" }]],
+  "minNodeVersion": "16.11.0"
```

Comment:
Aside: WDYT about a `minNodeExecVersion` option? So for old Node.js version can still transform the input and we can see if the output matches the fixture, but the test runner will ignore the `exec.js`. I guess most `minNodeVersion` can be replaced by `minNodeExecVersion`.

Reply:
Sounds good to me. I wonder if _any_ test needs to skip _compiling_ on old versions, or if we can just update `minNodeVersion` to only skip exeution.